### PR TITLE
ARTEMIS-910 Broker does not start if path contains spaces

### PR DESCRIPTION
**Pull request by gaohoward**

### CODE REVIEW

File: artemis-cli/src/main/resources/org/apache/activemq/artemis/cli/commands/bin/artemis.cmd [hunk 1]
```diff
@@ -58,9 +58,9 @@ call %ARTEMIS_INSTANCE%\etc\artemis.profile.cmd %*
 rem "Create full JVM Args"
 set JVM_ARGS=%JAVA_ARGS%
 if not "%ARTEMIS_CLUSTER_PROPS%"=="" set JVM_ARGS=%JVM_ARGS% %ARTEMIS_CLUSTER_PROPS%
-set JVM_ARGS=%JVM_ARGS% -classpath %ARTEMIS_HOME%\lib\artemis-boot.jar
-set JVM_ARGS=%JVM_ARGS% -Dartemis.home=%ARTEMIS_HOME%
-set JVM_ARGS=%JVM_ARGS% -Dartemis.instance=%ARTEMIS_INSTANCE%
+set JVM_ARGS=%JVM_ARGS% -classpath "%ARTEMIS_HOME%\lib\artemis-boot.jar"
```
Comment (clebertsuconic) [inline artemis-cli/src/main/resources/org/apache/activemq/artemis/cli/commands/bin/artemis.cmd]: did you actually test it on windows?


I over tested this on both linux, windows... there's a test under ./scripts validating it.. what is not working?

**Comment by clebertsuconic** [comment]
guys..please let me talk to howard before merging this.

**Comment by clebertsuconic** [comment]
@gaohoward @howardgao lets talk on monday... this is actually breaking things... the current version is actually working, and stopped working after your commit here. look at ./artemis-distribution/src/test/scripts/validate-spaces.sh (Just run the script)...

it works without your fix, and it fails with it.


Please, don't merge it.. I'm not sure what is the issue you found.

**Comment by gaohoward** [comment]
Hi Clebert, 

Sorry I didn't notice there is such a script. I just run the script and it passes on my machine (It doesn't give the final result but I can see it runs some tests and gracefully shutdown).

Regarding the testing, I actually tested on 2 platforms:

Ubuntu 16.04 LTS (64bit)
Windows 10 64bit (I didn't test cygwin thou).

I can confirm both platform works with my fix, but fails without.

Howard

**Comment by clebertsuconic** [comment]
@gaohoward I couldn't make it to fail, can you tell me how to replicate the issue?

In special I am looking to do it through the script test.

**Comment by gaohoward** [comment]
Hi Clebert,

It's strange that I couldn't make it either when I tried just now. I also test on windows it doesn't fail any more. I don't know why, maybe I messed up my environment.
Maybe we should just close this, it's a false alarm I guess.

Howard

**Comment by gaohoward** [comment]
Confirmed with @clebertsuconic, not an issue. Close it.

**Pull request closed without merging**